dev-python/ImageHash/ImageHash-3.4.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Image Hashing library"
+HOMEPAGE="https://github.com/JohannesBuchner/imagehash"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="test"
+
+RDEPEND="
```

Comment:
Yes, failed to add scipy. Fixed this and further directly bumped matplotlib2tikz to 0.6.7.